Week3/app.js

```diff
@@ -0,0 +1,239 @@
+   */
+  const people = [
+    {
+      name: 'Richie Baros',
+    },
+    {
+      name: 'Jayson Karner',
+    },
+    {
+      name: 'Dylan Wanner',
+    },
+    {
+      name: 'Travis Tarwater',
+    },
+    {
+      name: 'Angelique Lyall',
+    },
+    {
+      name: 'Stacie Dyar',
+    },
+  ];
+
+  people.sort(function(a, b) {
+    return a.name.localeCompare(b.name);
+  });
+  console.log(people);
+
+  /*
+   * Q2.
+   * Write a function that accepts array and returns only the even numbers in it.
+   * For example, if the input array is [13, 1, 4, 6, 8, 3, 2, 2], the output array should be [4, 6, 8, 2, 2].
+   */
+  let nes = [13, 1, 4, 6, 8, 3, 2, 2];
+  const newst = nes.filter(i => i % 2 === 0);
+  console.log(newst);
+
+  /*
+   * Q3.
+   * A class has 5 students defined below. Write a program to calculate average score of the class.
+   * Note: The average should be restricted to 2 decimal places.
+   */
+  const students = [
+    {
+      name: 'Student 1',
+      score: 93,
+    },
+    {
+      name: 'Student 2',
+      score: 87,
+    },
+    {
+      name: 'Student 3',
+      score: 84,
+    },
+    {
+      name: 'Student 4',
+      score: 90,
+    },
+    {
+      name: 'Student 5',
+      score: 96,
+    },
+  ];
+
+  let sum = 0;
+  for (let i = 0; i < students.length; i++) {
+    sum += parseInt(students[i].score, 10);
+  }
+  var avg = (sum / students.length).toFixed(2);
+  console.log(avg);
+
+  /*
+   * Q4.
+   * Write a JavaScript function which accept a string as argument and swaps the case of each character.
+   * For example if you input 'The Quick Brown Fox' the output should be 'tHE qUICK bROWN fOX'.
+   * Any character other than alphabets will stay as it is.
+   * Hint: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/charCodeAt
+   */
+
+  function newstring(str = '') {
+    let secondstring = '';
+    for (let i = 0; i < str.length; i++) {
+      if (str.charCodeAt(i) >= 97 && str.charCodeAt(i) <= 122) {
+        secondstring = secondstring.concat(str[i].toUpperCase());
+      } else if (str.charCodeAt(i) >= 65 && str.charCodeAt(i) <= 90) {
+        secondstring = secondstring.concat(str[i].toLowerCase());
+      } else {
+        secondstring = secondstring.concat(str[i]);
+      }
+    }
+    return secondstring;
+  }
+  console.log(newstring('The Quick Brown Fox'));
+}
+
+//More JavaScript
+
+// 1. Create a function that takes 3 arguments and returns the sum of the these arguments.
+
+function sum(arr) {
+  return arr.reduce(function(a, b, c) {
+    return a + b + c;
+  }, 0);
+}
+
+let numbers = [3, 4, 5];
+
+console.log(sum(numbers));
+
+// 2. Create a function named colorCar that receives a color, and prints out, 'a red car' for example.
+const colorCar = color => `a ${color} car`;
+console.log(colorCar('red'));
+
+// 3. Create an object and a function that takes the object as a parameter and prints out all of its properties and values.
+
+const carproperties = {
```

Comment:
I would advise to use `camelCase` here.

So instead of `carproperties ` I would do: `carProperties `